src/pelicanfs/core.py

```diff
@@ -47,6 +48,50 @@ class InvalidMetadata(PelicanException):
     No Pelican metadata was found for the federation
     """
 
+class _AccessResp:
```

Comment:
I found the top-level comments for other classes useful. Can you add one here, too?